src/main/controllers/style-guide/SjpPressListController.ts

```diff
@@ -94,10 +94,16 @@ export default class SjpPressListController {
 
     public async filterValues(req: PipRequest, res: Response): Promise<void> {
         if (validate(req.query?.artefactId as string)) {
+            const sjpPressMetaData = await publicationService.getIndividualPublicationMetadata(
```

Comment:
This method can technical return null if user doesn't have permission / not found which I think would then cause a crash on line 101

Reply:
Same comment applies to the public list controller too

Reply:
Updated the code to handle null metadata